.github/workflows/install.yml

```
@@ -111,14 +113,33 @@ jobs:
           echo "MAIN_BRANCH=${RAW_REF#refs/heads/}" >> $GITHUB_OUTPUT
           echo "MAIN_BRANCH=${RAW_REF#refs/heads/}"
       - name: Derive appropriate SHAs
+        if: ${{ inputs.run_merge_queue == false }}
         uses: nrwl/nx-set-shas@v4
         with:
           main-branch-name: ${{ steps.get-branch.outputs.MAIN_BRANCH }}
 
       - name: Set base/head sha output
+        if:
```

actionlint (1.7.4)

string should not be empty

(syntax-check)